composer.lock

```diff
@@ -347,16 +407,16 @@
         },
         {
             "name": "composer/composer",
-            "version": "1.10.x-dev",
+            "version": "dev-master",
```

Comment:
Should be rolled back, I guess you didn't `composer install` before running update?

Reply:
oh, right. Reverted the changes.